Index: twidere/build.gradle
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
====================================================================
--- twidere/build.gradle (date 1506175932000)
+++ twidere/build.gradle (revision )
@@ -20,7 +20,7 @@
        }


-       classpath "com.google.gms:google-services:${pluginsVersions['PlayServices']}"
+//     classpath "com.google.gms:google-services:${pluginsVersions['PlayServices']}"
    }
}
@@ -57,9 +57,9 @@
        productFlavors{
-           google{
-               dimension 'channel'
-           }
+//         google{
+//             dimension 'channel'
+//         }
        fdroid{
            dimension 'channel'
@@ -155,15 +155,15 @@
    implementation project(':twidere.component.nyan')

-   googleImplementation "com.google.android.gms:play-services-maps:${libVersions['PlayServices']}"
-   googleImplementation "com.google.android.gms:play-services-auth:${libVersions['PlayServices']}"
-   googleImplementation "com.google.android.gms:play-services-ads:${libVersions['PlayServices']}"
-   googleImplementation "com.google.maps.android-maps-utils:${libVersions['MapsUtils']}"
-   googleImplementation "com.anjlab.android.iab.v3:library:${libVersions['IABv3']}"
-   googleImplementation "com.dropbox.core:dropbox-core-sdk:${libVersions['DropboxCoreSdk']}"
-   googleImplementation("com.google.apis:google-api-services-drive:${libVersions['GoogleDriveApi']}")
-        exclude group: 'org.apache.httpcomponents'
-    }

+//   googleImplementation "com.google.android.gms:play-services-maps:${libVersions['PlayServices']}"
+//   googleImplementation "com.google.android.gms:play-services-auth:${libVersions['PlayServices']}"
+//   googleImplementation "com.google.android.gms:play-services-ads:${libVersions['PlayServices']}"
+//   googleImplementation "com.google.maps.android-maps-utils:${libVersions['MapsUtils']}"
+//   googleImplementation "com.anjlab.android.iab.v3:library:${libVersions['IABv3']}"
+//   googleImplementation "com.dropbox.core:dropbox-core-sdk:${libVersions['DropboxCoreSdk']}"
+//   googleImplementation("com.google.apis:google-api-services-drive:${libVersions['GoogleDriveApi']}")
+//        exclude group: 'org.apache.httpcomponents'
+//    }

        fdroidImplementation "org.osmdroid:osmdroid-android:${libVersions['OSMDroid']}"
@@ -299,5 +299,5 @@
 }

-apply plugin: 'com.google.gms.google-services'
+//apply plugin: 'com.google.gms.google-services'
\ No newline at end of file
